crypto/keys/mintkey/README.md

```diff
@@ -0,0 +1,36 @@
+Security parameter choice
+-------------------------
+
+The present Bcrypt security parameter used is 12, which should take about a quarter of a second on midrange consumer hardware (see [Benchmarking](#benchmarking) section below).
+
+For some background into security parameter considerations, see [here](https://auth0.com/blog/hashing-in-action-understanding-bcrypt/) and [here](https://security.stackexchange.com/questions/3959/recommended-of-iterations-when-using-pkbdf2-sha256/3993#3993).
+
+Given our security model, where an attacker would need to already have access to a victim's computer and copy the `~/.gaiacli` directory (as opposed to e.g. web authentication), this parameter choice seems sufficient for the time being.
+
+Benchmarking
+------------
+
+To run Bcrypt benchmarks:
+
+```bash
+go test -bench .
```

Comment:
Is this the actual command? I think we should have the relative path:

```shell
go test -v --bench github.com/cosmos/cosmos-sdk/crypto/keys/mintkey
```

Reply:
Sure; updated.